src/search/search-index-new/storage/types.ts

```diff
@@ -113,6 +113,11 @@ export interface ManageableStorage extends RegisterableStorage {
         filter: FilterQuery<T>,
         update,
     ): Promise<number>
+    suggest<T>(
+        collectionName: string,
+        filter: FilterQuery<T>,
+        opts?: FindOpts,
+    ): any
```

Comment:
This return type should at least be a `Promise` type, as it's going to be async. I think `Promise<string>` covers all uses

Reply:
That was a stupid suggestion: it would be `Promise<string[]>`. But anyway, I'll add these typings when I go to add in support for the assoc. suggestion PKs (as it will mean the return type can differ)